packages/commonwealth/scripts/gen-mermaid-erd.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+SQL=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )/mermaid-erd.sql
+MERMAID=$(PGPASSWORD=edgeware psql -h localhost -d commonwealth -U commonwealth -t -A -f $SQL)
+echo -e "# Autogenerated Mermaid ERD\n\n$(date)\n\n\`\`\`mermaid\n$MERMAID\n\`\`\`"
```

Comment:
Possible to add a "Last updated" before the timestamp, just to be super abundantly clear what the timestamp signifies?

Reply:
Makes sense!